fury/shaders/sdf/odf/sh_function.frag

```diff
@@ -0,0 +1,15 @@
+float calculateSH(int l, int m, in vec3 s)
+{
+    vec3 normS = normalize(s);
+    float thetaX = normS.z;
+    float phi = atan(normS.y, normS.x);
+    float result = calculateKFactor(l, abs(m)) * calcAssocLegendrePoly(l, abs(m), thetaX);
```

Comment:
I was using legacy basis so we need to adjust this and don't take the absolute value of `m`, so replace with:
`float result = calculateKFactor(l, m) * calcAssocLegendrePoly(l, m, thetaX);`

Reply:
Is that how it was in Dipy's codebase?